### DISCUSSION TITLE
Import PDF to an existing document

### DISCUSSION THREAD

**Discussion by cgloeckner** [xournalpp/xournalpp#5945] [Q&A] [answered]
Hi, importing a PDF as a new document works fine. But is there a way to import another PDF?
In other works: when taking notes on an empty page, then adding a work sheet PDF as second page to fill it out by hand, followed by more handwritten notes on a third (empty) page - all that in a single xournal++ file.

glocke

**Comment by PillowTrauma** [comment, marked as answer]
I don't think there's a way to have two separate PDFs associated with one Xournal file, but you can create a new page with a specific PDF page as the background. If you have everything you need beforehand, a workaround could be to use a tool like PDFGear to merge all your resources into one PDF, then pick the various worksheets as the background when inserting a new page.
